### DISCUSSION TITLE
Does syncoid let me setup incrementals versus existing snapshots on 2 systems that didn't use syncoid?

### DISCUSSION THREAD

**Discussion by drescherjm** [jimsalterjrs/sanoid#973] [Q&A] [answered]
I have recently added a new array on a new server (both Rocky linux with zfs-2.2.6) and used zfs send to send a snapshot (I manually created at that time) from the old server to the new server. Because of the data involved (and since I only have a 1 GBit network) this send and receive took over a week to transfer. Can I now install syncoid and have it periodically synchronize the incremental changes from the old system to the new. More data has been added to the original system. I eventually want to swap these and have the new as the one that the users use but for now we still use the old. I am running zfs-auto-snapshot on the old so that more snapshots have been added on that side. I believe I disabled these on the new side.

**Comment by jimsalterjrs** [comment, marked as answer]
Syncoid replicates all snapshots, regardless of what created them or what naming convention they follow.

**Comment by drescherjm** [comment]
Thanks a lot! Since this post I have done a lot of testing and I am impressed at how well this works. I should put these in production this week. I have over 300TB of zfs storage in 6 servers that need backups. Previously I was using LTO7 tape however our tape drive died just after the warranty expired and it didn't make a lot of sense to spend another 3k to 4k to replace it when I could use disks instead and have a much faster backup experience at a lower cost.
